### ISSUE TITLE
Path of Building won't get past an LUA error

### ISSUE THREAD

**Issue by Celcius345** [PathOfBuildingCommunity/PathOfBuilding#6394]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Screenshots of crash/graphical issue, version and options menu

![error](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/35319947/365236cc-919e-4fcd-8571-73792d08bb7f)
This happens when I start PoB every time and I cannot get past it. I am running as admin as well. 

### What are your system specifications and configuration?

Operating System: Windows 10 Home 64-bit (10.0, Build 19045) (19041.vb_release.191206-1406)
                 Language: English (Regional Setting: English)
      System Manufacturer: Gigabyte Technology Co., Ltd.
             System Model: X570 AORUS ELITE
                     BIOS: F38b (type: UEFI)
                Processor: AMD Ryzen 9 3900X 12-Core Processor             (24 CPUs), ~3.8GHz
                   Memory: 32768MB RAM
      Available OS Memory: 32690MB RAM
Card name: NVIDIA GeForce RTX 3070

### How to reproduce the issue

Launch path of building.

**Comment by Wires77** [comment]
Looks like a OneDrive issue.  Find your Settings.xml file in Documents/Path of Building, and open it.  That should force OneDrive to download it to your computer.  Launching PoB should work after that.  For the future, you can right click that file and select the option "Always keep available on this device"

**Comment by Jerbear0** [comment]
> Looks like a OneDrive issue. Find your Settings.xml file in Documents/Path of Building, and open it. That should force OneDrive to download it to your computer. Launching PoB should work after that. For the future, you can right click that file and select the option "Always keep available on this device"

I am having the same issue, i don't have OneDrive and my Documents folder is empty. I also tried to uninstall and reinstall it but no luck

**Comment by Jerbear0** [comment]
I tried with the portable version and i do not have this issue

**Comment by Celcius345** [comment]
Yup; same here. No OneDrive. It runs in Windows 95 compatibility mode, however, it runs very poorly. It runs so poorly in this mode that I haven't even bothered using it this league.

**Comment by flotos** [comment]
I have the same error that triggered when playing with support gems on my build, I can't start the app now as it automatically try to resume my build I suppose.

**Comment by Lilylicious** [comment]
You can delete settings.xml in your path of building folder, which should be in your documents folder, in order to prevent PoB from automatically opening your previous build. This will however also wipe your settings.

**Comment by Nthn0607** [comment]
> You can delete settings.xml in your path of building folder, which should be in your documents folder, in order to prevent PoB from automatically opening your previous build. This will however also wipe your settings.

@Lilylicious Deleting the file helped me fix it! Had the same issue and path of building couldn't load past the error screenshots above. Works as before now.

**Comment by Seawed** [comment]
If there is no desire to deal with one drive, then you can simply rename the file (like "settings2.xml"). Then the system will also consider it non-existent and the application will start.

**Comment by svnhddbst** [comment]
> You can delete settings.xml in your path of building folder, which should be in your documents folder, in order to prevent PoB from automatically opening your previous build. This will however also wipe your settings.

This was my problem and solution. I think I had something in a build setup wonky (didn't save the build so unsure at this time) involving a support gem, the build at that time had an alt quality support i believe. Because pob was set to resume where left off, it was trying to load improperly into an improperly set up state and crashing lua stuff.

**Comment by Lilylicious** [comment]
Note that it's possible to edit the file to remove the build to open on start, which would preserve the rest of the settings. If you're not comfortable with that though, deleting the entire file is your best bet so it's recreated.

**Comment by TurtleIs** [comment]
my PoB does not have a "settings,xml" file... how do I fix that?
